src/prisma/schema.prisma

```diff
@@ -162,14 +162,15 @@ model Transactions {
 }
 
 model Webhooks {
-  id        Int       @id @default(autoincrement()) @map("id")
-  name      String?   @map("name")
-  url       String    @map("url")
-  secret    String    @map("secret")
-  eventType String    @map("evenType")
-  createdAt DateTime  @default(now()) @map("createdAt")
-  updatedAt DateTime  @updatedAt @map("updatedAt")
-  revokedAt DateTime? @map("revokedAt")
+  id                    Int                     @id @default(autoincrement()) @map("id")
+  name                  String?                 @map("name")
+  url                   String                  @map("url")
+  secret                String                  @map("secret")
+  eventType             String                  @map("evenType")
+  createdAt             DateTime                @default(now()) @map("createdAt")
+  updatedAt             DateTime                @updatedAt @map("updatedAt")
+  revokedAt             DateTime?               @map("revokedAt")
+  ContractSubscriptions ContractSubscriptions[]
```

Comment:
This gets auto-added since I created an optional relation. I don't really like having it here.